spacemacs/keybindings.el

```diff
@@ -14,6 +14,10 @@
 (setq echo-keystrokes 0.02)
 ;; auto-indent on RET
 (define-key global-map (kbd "RET") 'newline-and-indent)
+;; reservable delete-other-windows
+(define-key global-map (kbd "C-x 1") 'zygospore-toggle-delete-other-windows)
+(define-key evil-window-map (kbd "o") 'zygospore-toggle-delete-other-windows)
+(define-key evil-window-map (kbd "C-o") 'zygospore-toggle-delete-other-windows)
```

Comment:
Consider to move these key bindings in the init function. This is required to be able to correctly exclude the package if needed.